Tasks/Butkevich/Task_2/Task_2.py

```diff
@@ -0,0 +1,108 @@
+                       "05": ["пять часов", "шестого", " пять минут", "без пяти минут"],
+                       "06": ["шесть часов", "седьмого", "шесть минут", "без шести минут"],
+                       "07": ["семь часов", "восьмого", "семь минут", "без семи минут"],
+                       "08": ["восьмь часов", "девятотого", "восемь минут", "без восьми минут"],
+                       "09": ["девять часов", "десятого", "девять минут", "без девяти минут"],
+                       "10": ["десять часов", "одинадцатого", "десять минут", "без десяти минут"],
+                       "11": ["одинадцать часов", "двенадцаого", "одинадцать минут", "без одинадцати минут"],
+                       "12": ["двенадцать часов", "первого", "двенадцать минут", "без двенадцати минут"],
+                       "13": ["один час", "второго", "тринадцать минут", "без тринадцати минут"],
+                       "14": ["два часа", "третьего", "четырнадцать минут", "без четырнадцати минут"],
+                       "15": ["три часа", "четвертого", "пятнадцать минут", "без пятнадцати минут"],
+                       "16": ["четыре часа", "пятого", "шестнадцать минут"],
+                       "17": ["пять часов", "шестого", "семнадцать минут"],
+                       "18": ["восемь часов", "седьмого", "Восемнадцать минут"],
+                       "19": ["семь часов", "восьмого", "девятнадцать минут"],
+                       "20": ["восемь часов", "девятого", "двадцать минут"],
+                       "21": ["девять часов", "десятого", "двадцать одна минута"],
+                       "22": ["десять часов", "одинадцатого", "двадцать две минуты"],
+                       "23": ["одинадцать часов", "двенадцатого", "двадцать три минуты"],
+                       "24": ["двенадцать часов", "первого", "двадцать четыре минуты"],
+                       "25": ["", "двадцать пять минут"],
+                       "26": ["", "двадцать шесть минут"],
+                       "27": ["", "двадцать семь минут"],
+                       "28": ["", "двадцать восемь минут"],
+                       "29": ["", "двадцать девять минут"],
+                       "31": ["", "тридцать одна минута"],
+                       "32": ["", "тридцать две минуты"],
+                       "33": ["", "тридцать три минуты"],
+                       "34": ["", "тридцать четыре минуты"],
+                       "35": ["", "тридцать пять минут"],
+                       "36": ["", "тридцать шесть минут"],
+                       "37": ["", "тридцать пять минут"],
+                       "38": ["", "тридцать восемь минут"],
+                       "39": ["", "тридцать девять минут"],
+                       "40": ["", "сорок минут"],
+                       "41": ["", "сорок одна минута"],
+                       "42": ["", "сорок две минуты", ],
+                       "43": ["", "сорок три минуты", ],
+                       "44": ["", "сорок четыре минуты"]
```

Comment:
a lot of duplicates, also I would recommend to move this declaration out of the while body, it will be more readable that way